subsys/net/l2/openthread/Kconfig.features

```diff
@@ -62,6 +62,36 @@ config OPENTHREAD_CHANNEL_MONITOR
 config OPENTHREAD_CHILD_SUPERVISION
 	bool "Enable child supervision support"
 
+config OPENTHREAD_MLE_INFORM_PREVIOUS_PARENT_ON_REATTACH
```

Comment:
All these Kconfigs should rather end up in `Kconfig.thread` instead of `Kconfig.features` - the latter covers features enabled by OT switches during the build, not the configuration.